src/screen.c

```diff
@@ -1667,6 +1667,8 @@ screen_start_highlight(int attr)
      */
     if (aep != NULL)
     {
+	if (aep->ae_u.cterm.font > 0 && aep->ae_u.cterm.font < 11)
```

Comment:
I've successfully tested the SRG20 (Gothic font) by extending the font limit by 1
```suggestion
	if (aep->ae_u.cterm.font > 0 && aep->ae_u.cterm.font < 12)
```